src/types/k8s.ts

```diff
@@ -44,7 +44,7 @@ export type K8sResourceCommon = K8sResourceIdentifier &
       finalizers: string[];
       generateName: string;
       generation: number;
-      labels: Record<string, string>;
+      labels?: Record<string, string>;
```

Comment:
from the screenshot provided by Cara in the ticket, it seems there's the possibility of the endpoint not return any `labels`, so I believe it would be safer to make this field "nullable" :)